description.md

```diff
@@ -17,22 +17,27 @@ The $\spose$ brings together people from around the world who work together to s
 It employs a decentralized governance structure, with individual projects retaining autonomy over their leadership, decision-making, and development processes.
 This community-driven approach allows for rapid innovation while seeking community consensus around ecosystem-wide decisions and standards.
 
-Consider a physicist studying particle collisions at the High Luminosity Large Hadron Collider (HL-LHC) to investigate fundamental questions about matter-antimatter asymmetry and the nature of dark matter.
-They might use IRIS-HEP's Elastic Analysis Facility, a cloud-based platform that provides a shared computing environment for high-energy physics researchers to process and analyze massive datasets, share code and results, and collaborate with colleagues globally.
-Then, to analyze collision data, the physicist might rely on Scikit-HEP (see Figure \ref{fig:scikit-hep}), a $\spose$ sub-ecosystem specifically designed for high-energy physics analysis \citep{Rodrigues:2020syo}.
-Scikit-HEP is built on a \emph{stack} of packages, including foundational scientific Python libraries like NumPy for numerical computing \citep{vanderwalt2011numpy, harris2020array}, SciPy for general scientific computing algorithms \citep{virtanen2020scipy}, Matplotlib for data visualization \citep{hunter2007matplotlib}, Pandas for data manipulation \citep{mckinney-proc-scipy-2010}, and Awkward Array for handling complex hierarchical data structures \citep{Pivarski_Awkward_Array_2018}.
-It also depends on domain-specific packages such as Uproot for reading ROOT files \citep{scikit-hep_uproot3_3_14_4}, Hist for advanced histogramming \citep{scikit-hep_boost_histogram_1_5_2}, Vector for manipulating physical vectors \citep{vector_v0_10_0}, and Particle \citep{scikit_hep_particle_v0_25_3} for accessing particle properties from the Particle Data Group database \citep{PhysRevD.110.030001}.
+Consider a physicist studying particle collisions at the Large Hadron Collider (LHC) to investigate questions about the interactions of matter and the fundamental forces of Nature and search for evidence of new physics.
+They might use a deployment of the Institute for Research and Innovation in Software for High Energy Physics \citep{Bockelman:2023gbj} (IRIS-HEP)'s Elastic Analysis Facility --- a cloud-based platform that provides shared computing environments for high energy physics (HEP) researchers to process and analyze massive datasets, share code and results, and collaborate with colleagues globally.
```

Comment:
Provide the definition of the IRIS-HEP acronym and cite the IRIS-HEP Strategic Plan.